easybuild/easyblocks/i/icc.py

```diff
@@ -74,21 +74,28 @@ def install_step(self):
                 'license_file_name': LICENSE_FILE_NAME_2012,
             }
 
-        super(EB_icc, self).install_step(silent_cfg_names_map=silent_cfg_names_map)
+        if LooseVersion(self.version) >= LooseVersion('2016'):
+            cfg_extras_map = {
+                'COMPONENTS': 'ALL',
+            }
+        super(EB_icc, self).install_step(silent_cfg_names_map=silent_cfg_names_map, silent_cfg_extras=cfg_extras_map)
 
     def sanity_check_step(self):
         """Custom sanity check paths for icc."""
 
         binprefix = "bin/intel64"
-        libprefix = "lib/intel64/lib"
+        libprefix = "lib/intel64"
         if LooseVersion(self.version) >= LooseVersion("2011"):
             if LooseVersion(self.version) <= LooseVersion("2011.3.174"):
                 binprefix = "bin"
-            elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
+            elif LooseVersion(self.version) >= LooseVersion("2013_sp1") and LooseVersion(self.version) < LooseVersion("2016"):
+                binprefix = "bin"
+                libprefix = "lib/intel64"
+            elif LooseVersion(self.version) >= LooseVersion("2016"):
                 binprefix = "bin"
-                libprefix = "lib/intel64/lib"
+                libprefix = "lib/intel64_lin"
```

Comment:
since the `binprefix` has remained the same, how about this:

```python
elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
    binprefix = "bin"
    if LooseVersion(self.version) >= LooseVersion("2016"):
        libprefix = "lib/intel64_lin"
```